stable/traefik/templates/configmap.yaml

```diff
@@ -77,3 +77,17 @@ data:
       {{- end}}
       {{- end}}
     {{- end }}
+    {{ if .Values.metrics.prometheus.enabled }}
+    [web.metrics.prometheus]
+      Buckets = {{ .Values.metrics.prometheus.buckets }}
+    {{- end }}
+    {{ if .Values.metrics.datadog.enabled }}
```

Comment:
i dont think we can configure more than one metrics provider per deployment? in that case i would preffer `else if` instead of `if`. What do you think?